Classes/Outline/NodeTextEditController.m

```diff
@@ -175,15 +175,15 @@ - (NSString*)unindentText:(NSString*)input {
     for (NSString *line in lines) {
         NSArray *captures = [line captureComponentsMatchedByRegex:@"( +).+"];
         if ([captures count] > 0) {
-            int spaces = [[captures objectAtIndex:0] length];
+            int spaces = (int)[[captures objectAtIndex:0] length];
             if (spaces < indentLevel)
                 indentLevel = spaces;
         }
     }
 
-    for (NSString *line in lines) {
-        // TODO: Do the unindention
-    }
+    // TODO: Do the unindention
```

Comment:
These 3 lines are legacy stuff from the original author so at this point I'm not sure of the idea behind the 'unindention'. Probably safe to just remove completely if you want